snuba/datasets/schemas/__init__.py

```diff
@@ -116,10 +121,21 @@ def get_column_differences(
 
             expected_type = self.get_columns()[column_name].type
 
+            if isinstance(expected_type, ReadOnly):
+                expected_type = expected_type.inner_type
+
+            # TODO: Remove if statement once modifiers are only used in migrations
             if column != expected_type:
-                errors.append(
-                    "Column '%s' type differs between local ClickHouse and schema! (expected: %s, is: %s)"
-                    % (column_name, expected_type, column)
-                )
+                while True:
```

Comment:
isn't this just 
```
col_type = column.get_raw() 
```

Reply:
Hmm, it could be but this way we're also validating that the nullability definition matches, not just the inner type